src/engraving/compat/midi/compatmidirender.cpp

```diff
@@ -932,7 +945,7 @@ void CompatMidiRender::createSlideOutNotePlayEvents(Note* note, NoteEventList* e
     int slideOn = NoteEvent::NOTE_LENGTH - totalSlideDuration;
     double velocity = !note->ghost() ? NoteEvent::DEFAULT_VELOCITY_MULTIPLIER : NoteEvent::GHOST_VELOCITY_MULTIPLIER;
     if (!hasTremolo) {
-        el->push_back(NoteEvent(0, onTime, slideOn, velocity, !note->tieBack()));
+        el->push_back(NoteEvent(0, onTime, slideOn - onTime, velocity, !note->tieBack()));
```

Comment:
I'm not 100% sure, but it seems NoteEvent m_len will always be 0 (slideOn-onTime). So this event makes no sense.